docs/tutorials/intro_tutorial.rst

```diff
@@ -118,47 +119,83 @@ call it)
 
 *Code will be added as the tutorial progresses.*
 
-Setting up the model
-~~~~~~~~~~~~~~~~~~~~
+Import Dependencies
+-------------------
 
-To begin writing the model code, we start with two core classes: one for
-the overall model, the other for the agents. The model class holds the
-model-level attributes, manages the agents, and generally handles the
-global level of our model. Each instantiation of the model class will be
-a specific model run. Each model will contain multiple agents, all of
-which are instantiations of the agent class. Both the model and agent
-classes are child classes of Mesa’s generic ``Model`` and ``Agent``
-classes. This is seen in the code with ``class MoneyModel(mesa.Model)``
-or ``class MoneyAgent(mesa.Agent)``. If you want you can specifically
-the class being imported by looking at the
-`model <https://github.com/projectmesa/mesa/blob/main/mesa/model.py>`__
-or
-`agent <https://github.com/projectmesa/mesa/blob/main/mesa/agent.py>`__
-code in the mesa repo.
-
-Each agent has only one variable: how much wealth it currently has.
-(Each agent will also have a unique identifier (i.e., a name), stored in
-the ``unique_id`` variable. Giving each agent a unique id is a good
-practice when doing agent-based modeling.)
-
-There is only one model-level parameter: how many agents the model
-contains. When a new model is started, we want it to populate itself
-with the given number of agents.
-
-The beginning of both classes looks like this:
+This includes importing of dependencies needed for the tutorial.
 
 .. code:: ipython3
 
     import mesa
 
+    # Visualization tools needed to explore the model.
+    import matplotlib.pyplot as plt
+    import numpy as np
+    import pandas as pd
+    import seaborn as sns
+
+Create Agent
+~~~~~~~~~~~~
+
+First create the agent. As the tutorial progresses, more functionality
+will be added to the agent.
+
+Background: Agents are the individual entities that act in the model.IT
+is a good modeling practice to make certain each Agent can be uniquely
+identified.
+
+Model Specific Information: Agents are the individuals that exchange
+money, in this case the amount of money an individual agent has is
+represented as wealth. Additionally, agents each have a unique
+identifier.
+
+Technical Details: This is done by creating a new class (or object) that
```

Comment:
I think it makes more sense to say "Code implementation" or "Python implementation". Probably the former.